src/backends/torch/torchutils.h

```diff
@@ -40,26 +40,50 @@ namespace dd
 
   namespace torch_utils
   {
+
+    /**
+     * \brief empty cuda caching allocator, not clear when useful
```

Comment:
This should be called after every job that allocates a model. Pytorch keeps cuda memory allocated event after the deletion of tensors, to avoid reallocating later. Sometimes it means reserving all the GPU ram even if training is over.

Reply:
ok